dns_entry_checker/dns_entry_checker.py

```diff
@@ -0,0 +1,127 @@
+import paramiko
+from re import compile
+from getpass import getpass
+from collections import defaultdict
+
+
+# Creates an SSH client through Paramiko with the users details and returns the client
+def create_client(host, user, password):
```

Comment:
Looks like you have some duplicate code here in #77 - this hints to me that perhaps we should combine your two scripts into a single package.

I'd say it's fine for now - but its worth looking at in the future.